build.gradle

```diff
@@ -271,6 +273,11 @@ project(':iceberg-bundled-guava') {
 }
 
 project(':iceberg-api') {
+  test {
+      if (JavaVersion.current() == JavaVersion.VERSION_17) {
```

Comment:
I think it would be great to define the JVM args in a single place so that all subprojects use them automatically when running with JDK17

Reply:
ACK

Reply:
Done